lib/rules/no-unused-vars.js

```diff
@@ -153,9 +150,32 @@ module.exports = {
         }
 
         /**
+         * Determine if an identifier is used either in for...of or for...in loops.
+         * @param {Reference} ref - The reference to check.
+         * @returns {Boolean} whether reference is used in the for...of or for...in loops
```

Comment:
What spec, are we talking about? I looked at http://usejsdoc.org/tags-returns.html.

Is there a ticket to include this in http://eslint.org/docs/rules/valid-jsdoc ?